mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -491,5 +497,17 @@ public void testCreateTableWithNotSupportedTypes() {
           }
       );
     }
+
+    shell.setHiveSessionValue(InputFormatConfig.SCHEMA_AUTO_CONVERSION, "true");
```

Comment:
I'd probably make this a separate test case. No need to tack it on to the UnsupportedTypes test.

Reply:
Moved to a separate test case.